scripts/barclamp_mgmt_lib.rb

```diff
@@ -260,6 +260,8 @@ def generate_assets_manifest
 
   manifests = Pathname.new(CROWBAR_PATH).join("barclamps", "manifests")
 
+  manifests.mkdir unless manifests.exist?
```

Comment:
Which manifest file cannot be written? We're just reading files from the directory here (and I suggested to skip that step if there's no directory). Or do you mean the one in /public/assets/?